models/amazon_ads__portfolio_report.sql

```diff
@@ -2,7 +2,7 @@
 
 with report as (
     select *
-    --use campaign report since portfolio report not provided
+    --use campaign report since portfolio report not provided 
```

Comment:
This inline comment was helpful to me when reviewing this model. However, I think this will be lost for users of the package that don't dive deep into the models.

Would you be able to extrapolate this inline comment (and your findings around the behavior of the portfolio in relation to the account) and add it to a DECISIONLOG.md entry for this package.

No need to have that be part of this task. The DECISIONLOG.md entry can be part of the final documentation task for this package.

Reply:
Done. I started some notes in a decision log with a note to update it later.